### PR TITLE
Logo Added on website

### PR DESCRIPTION
**Pull request by sarthaksavvy**
| Q                       | A
| ---------------| ---
| Bug fix?            | no
| New feature?   | yes
| Fixed tickets    | #223 

### New logo added with the old image

### CODE REVIEW

**Comment by Jibbarth** [comment]
Hey @sarthaksavvy Thanks for working on this :+1: 

Is it possible to reuse images in `art` folder instead of having duplication ? We do this for the changelog with composer task ?

**Comment by sarthaksavvy** [comment]
actually, it was difficult to use two different images in vuepress, that's why I have merged them.

But I will try it again and let you know.

**Comment by olivernybroe** [comment]
Thanks! Nice to have this updated 😍

@sarthaksavvy I think the easiest way to get it working is by doing the same as we did with changelog. 
https://github.com/nunomaduro/phpinsights/blob/f6597e8bb362a93e07e147d9b578c8fbd9ab0194/composer.json#L65

Basically before the website is deployed, we run this composer command and it copies the file into the vuepress directory.

Ofc. The best solution would be to get vuepress to read outside it's own folder or to use a symbolic link correctly, but the copy approach is fine for us.

**Comment by nunomaduro** [comment]
cc @sarthaksavvy

**Comment by sarthaksavvy** [comment]
Moved logo to art directory and moving it to vuepress when we run composer script.

I tried for reusing images from art directory but can't find how to use two images in vuepress.

File: composer.json [hunk 1]
```diff
@@ -3,12 +3,10 @@
     "description": "Instant PHP quality checks from your console.",
     "keywords": ["php", "insights", "console", "quality", "source", "code"],
     "license": "MIT",
-    "authors": [
-        {
-            "name": "Nuno Maduro",
-            "email": "enunomaduro@gmail.com"
-        }
-    ],
+    "authors": [{
```
Comment (olivernybroe) [inline composer.json]: Could you change this style back?

**Comment by nunomaduro** [comment]
Thanks!

**Comment by sarthaksavvy** [comment]
Image link is broken on the website. What's the issue? Script not working?

**Pull request merged**